templates/framework/doc_tree/_load_external_document_component.html.twig

```diff
@@ -9,9 +9,11 @@
             </div>
         </div>
     </div>
-    
-    <p style="margin-top:25px"><b>Example:</b> To load the Common Core ELA standards from PCG’s OpenSALT™ server, you can use the following url:</p>
+
+    <p style="margin-top:25px"><b>Example:</b> To load the Mathematics: Georgia Standards of Excellent standards from Georgia's OpenSALT™ server, you can use the following url:</p>
     <ul>
-        <li>https://opensalt.net/ims/case/v1p0/CFPackages/2b8afaec-12a7-11e8-9ed1-0242ac120005</li>
+        <li>https://case.georgiastandards.org/ims/case/v1p0/CFPackages/23a8e45a-9d5a-11e7-81bc-064e21a83c7c.json</li>
```

Comment:
The `.json` is not needed and should probably not be part of the API URL (by the spec).